### DISCUSSION TITLE
Actual Frame Rate

### DISCUSSION THREAD

**Discussion by chaudhuri-ab** [gopro/labs#447] [Q&A] [answered]
Just curiously when shooting 30FPS on a Hero Black 9 what is the actual frame rate it saves the MP4 to if I want to Time Code Sync it?

It seems that at 30 FPS and 60 FPS the go pros drop down to 29.97 and 59.94, respectively with drop frame time coding according to the go pro site.

[Link](https://community.gopro.com/s/article/Drop-Frame-Timecode-Information?language=en_US#:~:text=Question-,Do%20GoPro%20cameras%20support%20drop%20frame,Where%20does%20it%20apply%3F&text=The%20cameras%20support%20true%2024p,timecode%20of%2029.97%20and%2059.94.)

Is that the case and if so, when using an external Time Code Generator, would I go with 29.97 drop frame encoding to match the go pro at 30 FPS or is the go pro actually not dropping a frame in time coding?

**Comment by dnewman-gpsw** [comment, marked as answer]
Drop-frame timecode is not related to capture frame rate. SMPTE standard for video is always at 29.97 frames per second (technically 3000/1001 fps.)  This creates a weird scenario where time in time-code runs point one percent slower than reality.  xx:xx:01:xx - the seconds digit changes every 1.001s, this is correct and normal.  Dropframe timecode was introduced to re-sync timecode with real-time, but it is less often used as it is not continuous.  In normal non-drop timecode:  
```
  01:08:59:28
  01:08:59:29
  01:09:00:00
  01:09:00:01 
```
whereas drop-frame would be:
```
  01:08:59:28
  01:08:59:29
  01:09:00:02 
  01:09:00:03
```

Every minute two frame are skipped, except for every ten minute.  It is like a leap year system.

GoPro uses frame since midnight timecode, which is inherently non-drop (although all timecode can be converted in post.)  

Always use non-drop is my recommendation.
